airbyte-integrations/connector-templates/source-singer/source_{{snakeCase name}}_singer/source.py.hbs

```diff
@@ -30,7 +30,7 @@ from base_singer import SingerSource, Status
 
 
 class Source{{properCase name}}Singer(SingerSource):
-    TAP_CMD = "tap-{{properCase name}}"
+    TAP_CMD = "{{ tap_name }}"
```

Comment:
this is not always the case e.g: sometimes pypi package `pipelinewise-tap-google-analytics` is called via `tap-google-analytics`

Reply:
same. My goal isn't to fix something that wasn't working and that can't really be "templated". I just want to enable automatic testing